server/neptune/workflows/internal/root/deployment.go

```diff
@@ -0,0 +1,13 @@
+package root
```

Comment:
Why did you decide to put this here?

Reply:
I originally added it to the terraform package but it creates a cyclic dependency. Since a deployment is a root's deployment technically, I figured `root.Deployment `would give better context to the caller. So, I chose to move it to the root package. Do you have any suggestion?